lib/auth_web/controllers/role_controller.ex

```diff
@@ -0,0 +1,104 @@
+defmodule AuthWeb.RoleController do
```

Comment:
Again, does there need to be some authentication here so everyone can't create and delete roles?

Reply:
Related to #99 once the helper function is in place we can restrict everything. 👍

Reply:
This is still a security hole, issue in #116

Reply:
@th0mas agree that this is a _potential_ security issue so I have addressed it. https://github.com/dwyl/auth/issues/116#issuecomment-694113729

Please take a look and LMK (though comments on the issue) what is still required. Thanks.